src/inspector_socket.cc

```diff
@@ -48,16 +49,17 @@ static void dump_hex(const char* buf, size_t len) {
 }
 #endif
 
+static void remove_from_beginning(std::vector<char>* buffer, size_t count) {
```

Comment:
I think @eugeneo is following [this rationale](https://google.github.io/styleguide/cppguide.html#Reference_Arguments) which I subscribe to as well (I might be biased though :).